games/minimal/mods/testentities/armor.lua

```diff
@@ -0,0 +1,41 @@
+-- Armorball: Test entity for testing armor groups
+-- Rightclick to change armor group
+
+local phasearmor = {
+	[0]={icy=100},
+	[1]={firy=100},
```

Comment:
`[1]` .. `[4]` (everything except `[0]`) can be omitted. Might be more readable this way, though.